service/cloudconfigv3/template.go

```diff
@@ -1,8 +1,48 @@
 package cloudconfigv3
 
+import (
+	"bytes"
+	"html/template"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/randomkeys"
+)
+
 const (
 	set_ownership_etcd_data_dir_dropin = `[Unit]
 Requires=etc-kubernetes-data-etcd.mount
 After=etc-kubernetes-data-etcd.mount
+`
+	encryptionConfigTemplate = `
+kind: EncryptionConfig
+apiVersion: v1
+resources:
+  - resources:
+    - secrets
+    providers:
+    - aescbc:
+        keys:
+        - name: key1
+          secret: {{.EncryptionKey}}
+    - identity: {}
```

Comment:
It looks like it should be added to k8scloudconfig {master,worker}_template.go files, and adding EncryptionKey to Params struct.

Reply:
I will add it in the 2_0_0 version that is currently used nowhere

Reply:
https://github.com/giantswarm/k8scloudconfig/pull/269

Reply:
It will be part of 3_0_0. lets this one on 2_0_0 please

Reply:
Fine with me.